spring-cloud-dataflow-apps-generator-plugin/src/main/java/org/springframework/cloud/dataflow/app/plugin/generator/AppDefinition.java

```diff
@@ -213,8 +223,7 @@ public static class ContainerImage {
 		private ContainerImageFormat format = ContainerImageFormat.Docker;
 
 		/**
-		 * True will attempt to inline the (white) filtered Spring Boot metadata as Base64 encoded
-		 * property.
+		 * True will attempt to inline the (white) filtered Spring Boot metadata as Base64 encoded property.
```

Comment:
I know this was already in here, but do you mind removing the non-inclusive legacy terminology from this comment ("(white)")?